_posts/2017-06-02-jslang-meetup-async-await.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title:  "Async Function Kata at #jslang Meetup"
+date:   2017-06-02 08:03:00 +0200
+categories: meetup javascript kata
+---
+
+The callback-hell times are long over. We have better tools now. But which of the new kids on the block do you want to 
+play with? In this meetup, last night we tried out [async functions][async-funcs], also known by the name "async/await". 
+It was a fun learning experience. Keep reading and get inspired.
+
+## The meetup
+["JavaScript The Language" (#jslang)][jslang-meetup] a hands-on meetup only about JavaScript the language. Not about any library, 
+framework or the latest programming technique. You could argue that the language JavaScript has become kind of a 
+framework itself, you need(ed) a transpiler, the language changes as fast as those hipster frameworks. 
+Let's put that aside.
+We focus on language features only. We try to get a handle on a specific one at a time and sharpen this one 
+feature for our tool belts. That's why this is a very hands-on meetup.
+Also there is no talks or alike, just a short intro and the crowd works together. All attendees are actively 
+participating, by coding, discussing or facilitating.
+You want to do that too, in your city? Just start a meetup, take our katas that we solve and make it happen.
+
+## Our Scope and the Kata
+I announced the meetup about two weeks ago. There was no #jslang meetup since April 2016, more than one year ago. 
+But since I joined HolidayCheck in May 2017 and it is in my DNA to promote things I enjoy, I had to do restart this 
+meetup. The language JavaScript is moving forward and we need to keep up with what it offers.
+Async functions are one of the most interesting things that is changing in the language. In order to understand 
+them in the JS context I prepared a [short slide set sumarizing the state the async functions feature][slides]. 
+That was all of presentation we had. I set the context and introduced [the kata][kata] that we want to work on. 
+In the kata we want to combine two asynchronous functionalities, 1) getting the geolocation and 2) loading the 
+(nearest) airports depending on that result.
+The interesting part is actually not (only) the implementation but it already starts with the tests. Have you ever 
+had trouble writing asynchronous tests? Do this kata and practice even more!
+
+## How we did it
+We were twelve people at the event. And a quick hand-sign and we all agreed on setting up two mobs and solve the 
```

Comment:
I guess you meant "after a quick vote by show of hands" ? Maybe you could also make _"mob programming"_ a link to wikipedia for those who might not know what it is. From that point of view it might even be preferable to have the word "mob programming" (as a link) occur before "setting up to mobs". Just a thought.